pkg/netconf/nftables.go

```diff
@@ -44,8 +56,9 @@ type (
 // NewNftablesConfigApplier constructs a new instance of this type.
 func NewNftablesConfigApplier(kb KnowledgeBase, validator net.Validator) net.Applier {
 	data := NftablesData{
-		Comment: versionHeader(kb.Machineuuid),
-		SNAT:    getSNAT(kb),
+		Comment:      versionHeader(kb.Machineuuid),
+		SNAT:         getSNAT(kb),
+		DNSProxyDNAT: getDNSProxyDNAT(kb, dnsPort),
```

Comment:
It should be configurable by flag whether or not to proxy dns.
`metal-networker` should not set this option ... otherwise plain metal machines will not have DNS access and kubernetes workers will not get into ready state.

When used as library from the `firewall-controller` DNAT for DNS should be added.
This means that `firewall-controller` needs also to be adapted to render the nf table `metal`.

Reply:
From what i was able to find, firewall controller [reconciles only frr config](https://github.com/metal-stack/firewall-controller/blob/master/pkg/network/network.go#L28). So we need to update firewall controller to generate nftables rules(from networker) too?

Reply:
Firewall controller renders `/etc/nftables/firewall-controller.v4`
It should also reconcile `/etc/nftables/rules` - as metal-networker does.